packages/lodestar/src/network/gossip/gossip.ts

```diff
@@ -193,8 +191,27 @@ export class Gossip extends (EventEmitter as { new(): GossipEventEmitter }) impl
     }
   }
 
-  private registerHandlers(): Map<string, GossipHandlerFn> {
-    const forkDigest = this.chain.currentForkDigest;
+  private handleForkDigest = async (forkDigest: ForkDigest): Promise<void> => {
+    const forkDigestHash = toHexString(forkDigest).toLowerCase().substring(2);
```

Comment:
Minor thing, I think we should just use `toHexString(forkDigest)` or `toHex` from `lodestar-utils` for display, without lower casing or substringing.